src/server/pachyderm_test.go

```diff
@@ -10344,6 +10344,30 @@ func TestPipelineAncestry(t *testing.T) {
 	checkInfos(infos)
 }
 
+func TestConsistingHashingPFS(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+	t.Parallel()
+	c, _ := minikubetestenv.AcquireCluster(t, minikubetestenv.WithPachdReplicas(2))
+	numTestRepos := 50
+	eg, _ := errgroup.WithContext(context.Background())
+	for i := 0; i < numTestRepos; i++ {
+		eg.Go(func() error {
+			require.NoErrorWithinT(t, time.Second*60, func() error {
+				repo := tu.UniqueString(t.Name())
+				require.NoError(t, c.CreateRepo(repo))
+				commit, err := c.StartCommit(repo, "master")
+				require.NoError(t, err)
+				require.NoError(t, c.PutFile(commit, "file", strings.NewReader("foo"), client.WithAppendPutFile()))
```

Comment:
You can use a one-off put file for this. Just a put file without start / finish commit will automatically start and finish a commit for the put file operation. Also, you need to wait for the commit to be finished because finish commit only marks it as finishing. The PFS master needs to pick up the commit and finish it for it to transition to the finished state.